src/listeners/member-verification/modal-submit-validator.ts

```diff
@@ -113,9 +113,19 @@ export class InteractionCreateListener extends Listener<
       teamObject = teams;
     }
 
+    const guildSettings = await settingsManager.get(interaction.guildId);
+    const guild = await interaction.client.guilds.fetch(interaction.guildId);
+
+    const verifiedRole = guildSettings?.verifiedRole;
+    if (!verifiedRole) {
+      return;
+    }
+
     let locationRole = "";
     const rolesDictionary = Object.fromEntries(
-      interaction.guild?.roles.cache.map(role => [role.name.toLowerCase(), role.id]) ?? []
+      interaction.guild?.roles.cache
```

Comment:
Nit: Could use `guild`, since we already fetched it.
```suggestion
      guild.roles.cache
```